src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colSup = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColour(colSup.getRandomColor().toString());
+        ball.setNumber(random.nextInt(100));

```

Comment:
100 - is a magic number. Make it constant.

Reply:
Created  "public static final int MAX_BALL_NUMBER = 100;"
Is that correct?

Reply:
yes